package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cliparse",
-  "version": "0.2.5",
+  "version": "0.2.6",
```

Comment:
can you remove this change from the PR please? I'll bump the version in a separate commit.